frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -482,6 +516,22 @@ export const getExternalImagelValues = (appResource: K8sResourceKind) => {
   };
 };
 
+export const getFileUploadValues = (resource: K8sResourceKind, buildConfig: K8sResourceKind) => {
+  const resourceName = resource.metadata.name;
+  const fileName = buildConfig.metadata?.annotations?.jarFileName ?? '';
+  const javaArgs: string = resource.spec?.template?.spec?.containers
+    ?.find((container) => container.name === resourceName)
+    ?.env?.find((args) => args.name === 'JAVA_ARGS').value;
```

Comment:
I think this will fail for a scenario where Env var name is not 'JAVA_ARGS' and will would acc `values`. Observed this when i added env say `TEST: test`while edit flow 

![image](https://user-images.githubusercontent.com/5129024/112834240-62713a00-90b5-11eb-8047-92daebb92f92.png)